src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -81,9 +97,29 @@ const AnnotationActivity = ({
     const activityLinkMessage = isFileVersionUnavailable
         ? messages.annotationActivityVersionUnavailable
         : { ...linkMessage, values: { number: linkValue } };
+    const clickHandlers = useSuppressiveClicks<HTMLDivElement>({ onClick: handleOnSelect });
+    const interactiveDivProps = isInteractive
+        ? {
+              role: 'button',
+              tabIndex: '0',
+          }
+        : null;
+
+    React.useEffect(() => {
+        setIsInteractive(!(isMenuOpen || isEditing || isFileVersionUnavailable));
+    }, [isEditing, isFileVersionUnavailable, isMenuOpen]);
 
     return (
-        <div className="bcs-AnnotationActivity" data-resin-feature="annotations">
+        <div
+            className={classNames('bcs-AnnotationActivity', { 'is-interactive': isInteractive })}
```

Comment:
We'll want to prefix all class names to avoid collisions with other consuming applications.